build.sbt

```diff
@@ -44,7 +44,8 @@ inThisBuild(Def.settings(
   testOptions += Tests.Argument(TestFrameworks.JUnit, "-q", "-v"),
   Dependencies.Versions,
   Formatting.formatSettings,
-  shellPrompt := { s => Project.extract(s).currentProject.id + " > " }
+  shellPrompt := { s => Project.extract(s).currentProject.id + " > " },
+  concurrentRestrictions in Global += Tags.limit(Tags.Test, 1)
```

Comment:
yeah sounds good, until we stabilize things; weird that it started failing so much on our infra recently after the upgrades

Reply:
No, it runs only on my machine right now. I guess the much bigger amount of tests after the client pool merges changed the execution order in a way that makes it more likely somehow.